index.bs

```diff
@@ -909,7 +972,49 @@ partial interface HTMLIFrameElement {
       <li>Return "<code>Disabled</code>".</li>
     </ol>
   </section>
+  <section>
+    <h3 id="report-violation">Generate report for violation of |feature| policy
+    with |message| and |disposition|, on |settings|</h3>
+
+    <p> Given a feature (|feature|), a string (|message|), another string
+    (|disposition|), an <a>environment settings object</a> (|settings|), and an
```

Comment:
I think that |disposition| should be left out of the parameter list here. Even if/when we do have a report-only mode defined, the spec calling this won't know if it's active or not. This algorithm should eventually be responsible for determining the state of that flag and setting the report body appropriately.